pilot/pkg/model/authentication.go

```diff
@@ -97,13 +110,75 @@ func (policy *AuthenticationPolicies) addRequestAuthentication(configs []Config)
 	}
 }
 
+func apiModeToMutualTLSMode(mode v1beta1.PeerAuthentication_MutualTLS_Mode) MutualTLSMode {
+	switch mode {
+	case v1beta1.PeerAuthentication_MutualTLS_DISABLE:
+		return MTLSDisable
+	case v1beta1.PeerAuthentication_MutualTLS_PERMISSIVE:
+		return MTLSPermissive
+	case v1beta1.PeerAuthentication_MutualTLS_STRICT:
+		return MTLSStrict
+	default:
+		return MTLSUnknown
+	}
+}
+
 func (policy *AuthenticationPolicies) addPeerAuthentication(configs []Config) {
+	foundNamespaceMTLS := make(map[string]v1beta1.PeerAuthentication_MutualTLS_Mode)
+
 	for _, config := range configs {
 		policy.peerAuthentications[config.Namespace] =
 			append(policy.peerAuthentications[config.Namespace], config)
+
+		// Mesh & namespace level policy must be named as `default` (config_consts.DefaultAuthenticationPolicyName)
+		// per validation rule. Extra check for empty selector is optional.
```

Comment:
are you saying the the name equals to `default` is enough? and we don't have to check selector?

Reply:
With validation on, it should be equivalent. But thinking a bit more on this, may be we should check both just to be safe.

Reply:
yes. it's building on some convention, (you could say that's part of API), but checking selector seems more robust.